app/models/book.rb

```diff
@@ -5,12 +5,12 @@ class Book < ApplicationRecord
 
   validates_presence_of :title, :author, :reference_number, :book_type
   validates :cover_image,
-            attached: true,
-            dimension: {
-              width: { min: 100, max: 300 },
-              height: { min: 100, max: 300 },
-              message: 'is not given between dimension. Height between 100 and 300, width between 100 and 300.'
-            }
+            attached: true
+            # dimension: {
+            #   width: { min: 100, max: 300 },
+            #   height: { min: 100, max: 300 },
+            #   message: 'is not given between dimension. Height between 100 and 300, width between 100 and 300.'
+            # }
```

Comment:
Good practice to remove rather than commenting out. If it's commented out, then you clearly don't need it to be in the codebase